### PR TITLE
Embed video on front page

### PR DESCRIPTION
**Pull request by petterarvidsson**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
                     <div class="col-md-8 col-md-offset-2">
                         <h1 class="brand-heading">Konstructs</h1>
                         <p class="intro-text">An open source Infiniminer/Minecraft inspired multiplayer game.</p>
+                        <p>
+                          <iframe width="280" height="158" src="https://www.youtube.com/embed/8HmZ1Rk0Kak" frameborder="0" allowfullscreen></iframe>
```
Comment (nsg) [inline _includes/header.html:10]: The static size 280x158px will be problematic on smaller screens (cell phones).
Maybe something like `style="max-width: 280px; height: 158px"` ... or some thing similar.

Comment (petterarvidsson) [inline _includes/header.html:10]: Aha, I was just copying the stuff from youtube :-) Will try that one.

Comment (petterarvidsson) [inline _includes/header.html:10]: Seems very tricky to make a iframe resize after the content is loaded. What you recommended didn't work. Without the video I can not make the page smaller than 320 and with the video it is the same. Maybe it doesn't impact the minimal width and it is rather dictated elsewhere?

Comment (petterarvidsson) [inline _includes/header.html:10]: It's looking "good" in portrait mode on my HTC Magic. I'd say few phones have lower resolution than that :-)

**Pull request merged**